docs/schema.md

```diff
@@ -241,6 +245,39 @@ Databases are encouraged not to include those in the first place. (The goal is
 to balance flexibility of presentation with not exposing vulnerability database
 display sites to unnecessary vulnerabilities.)
 
+## severity field
+
+```json
+{
+	"severity": [ {
+		"type": string,
+		"score": string
+	}
+} ]
+```
+
+The `severity` field is a JSON array that allows generating systems to
+describe the severity of a vulnerability using one or more scoring
+methods. Each `severity` item is a JSON object specifying a `type` and
+`score` property, described below.
+
+### severity[].type field
+
+The `severity[].type` property must be one of the types defined below,
+which describe how the `associated` score was derived. This allows
+processing systems to decide how to handle each severity.
+
+| Severity Type | Score Description |
+| --------- | ----------- |
+| `CVSS_V3` | A computed score representing the unique characteristics and severity of the vulnerability using a version of the [Common Vulnerability Scoring System notation](https://www.first.org/cvss/) that is >= 3.0 and < 4.0. |
+| `PLAIN_TEXT` | A human-derived and human-readable rating of the severity, based on interpretation of available vulnerability information. |
```

Comment:
Could we define this a bit more strictly? Is the intention to have a human readable vulnerability metric, i.e. "Critical", "High", "Medium", "Low"? If so, I think it would be better to state these here so that these entries are consistent across all vuln DBs.